app.ts

```diff
@@ -0,0 +1,295 @@
+window.onload = async function () {
+  $("#loader").hide();
+  await fetchTotalRecordCount();
+  updateRecordsPerPage();
+  fetchAndDisplayColumns();
+  handleSpecialResize();
+};
+// Global variables
+const IMQS: string = "http://localhost:2050";
```

Comment:
camelCase please

Reply:
Please ignore, as for global constant values we are supposed to always use all caps.